src/core/lib/experiments/config.cc

```diff
@@ -111,11 +136,22 @@ void TestOnlyReloadExperimentsFromConfigVariables() {
   PrintExperimentsList();
 }
 
-bool IsExperimentEnabled(size_t experiment_id) {
-  // Normal path: just return the value;
+void LoadTestOnlyExperimentsFromMetadata(
+    const ExperimentMetadata* experiment_metadata, size_t num_experiments) {
+  g_test_experiments =
+      new TestExperiments(experiment_metadata, num_experiments);
+}
+
+template <>
```

Comment:
Consider using `IsExperimentEnabled` and `IsTestExperimentEnabled` instead. The templates obscure the fact that `IsExperimentEnabled<true>` is a test-only function, there are no other generic types here that would benefit from using templates, and all call sites know precisely which version they want to call.

Reply:
Thanks. Fixed it.